cache/config.go

```diff
@@ -0,0 +1,41 @@
+package cache
+
+import "time"
+
+type Config struct {
+	ChannelSize    int
```

Comment:
`ChannelSize` doesn't mean anything to caller. For me it would be something like `PerWatchBufferSize` vs `SharedBufferSize` below